src/index.js

```diff
@@ -0,0 +1,300 @@
+let Cents;
+const { BigNumber } = require("bignumber.js");
+
+// http://stackoverflow.com/questions/3885817/how-to-check-if-a-number-is-float-or-integer
+const isInt = (maybeInt) => maybeInt % 1 === 0;
+
+module.exports =
+  Cents =
+  Cents =
+    (function () {
+      let compareCentsFunction = undefined;
+      let comparators = undefined;
+      Cents = class Cents {
+        static initClass() {
```

Comment:
oof, could we clean this up? this is crazy! let's make this a regular `class` without all this indirection?